practice_app/components/MapView.module.css

```diff
@@ -1,5 +1,5 @@
 .mapContainer {
     position: relative;
-    width: 100vw;
```

Comment:
why remove width? I think it looks cool when it wide all over the page :D

Reply:
Ok, we can set width again. We only apply margin to text part then.

Reply:
Okay can you revert the width change please? Then we can accept this PR I think